settings/src/components/AuthSecurityPassword.vue

```
@@ -0,0 +1,135 @@
+    name: "AuthSecurityPassword",
+    props: {
+        tokens: {
+            type: Array,
+            requried: true,
+        },
+    },
+    components: {Password},
+    data() {
+        return {
+            baseurl: OC.generateUrl('/settings/personal/changepassword'),
+            oldpass: '',
+            newpass: '',
+            showloader: false,
+            score: 0,
+            scoreMessages: ['Very weak password',
+            'Weak password',
+            'So-so password',
+            'Good password',
+            'Strong password']
+        }
+    },
+    computed: {
+        isOpen: function() {
+            return this.newpass != '';
+        }
+    },
+    methods: {
+        getScore (score)
+        {
+            this.score = score;
+        },
+        scoreMessage() {
+            return this.scoreMessages[this.score];
+        },
+        changePassword()
+        {
+            if (this.oldpass == this.newpass || this.newpass == '')
+            {
+                OC.msg.finishedSaving('#password-error-msg',
+				{
+                    'status' : 'error',
+					'data' : {
+						'message' : t('settings', 'Unable to change password')
+					}
+                });
+                return false;
+            }
+
+            this.showloader = true;
+            var formParameters = {
+                'oldpassword' : this.oldpass,
+                'newpassword' : this.newpass,
+                'newpassword-clone' : this.newpass
+            };
+            var headers = {
+                contentType : 'application/x-www-form-urlencoded; charset=UTF-8'
+            }
+
+            return Axios.post(this.baseurl,formParameters,{headers: headers}).then(
+                function(resp) {
+                    console.log(resp);
+                    OC.msg.finishedSaving('#password-error-msg', resp.data);
+            })
+            .then(tap(() => this.showloader = false))
+            .catch(tap(() => this.showloader = false))
```

this will discard the error. `tap` does not work with errors. move the `this.showloader=false` to the other `catch` callback below

---

Fixed!